src/ck-ldb/CentralLB.C

```diff
@@ -107,9 +99,9 @@ void CentralLB::initLB(const CkLBOptions &opt)
   if (_lb_predict) predicted_model = new FutureModel(_lb_predict_window);
   else predicted_model=0;
   // register user interface callbacks
-  theLbdb->SetupPredictor((LDPredictModelFn)(staticPredictorOn),(LDPredictWindowFn)(staticPredictorOnWin),(LDPredictFn)(staticPredictorOff),(LDPredictModelFn)(staticChangePredictor),(void*)(this));
+  lbmgr->SetupPredictor((LDPredictModelFn)(staticPredictorOn),(LDPredictWindowFn)(staticPredictorOnWin),(LDPredictFn)(staticPredictorOff),(LDPredictModelFn)(staticChangePredictor),(void*)(this));
 
-  myspeed = theLbdb->ProcessorSpeed();
+  myspeed = lbmgr->ProcessorSpeed();
```

Comment:
Is proc speed something that would be stored in the LBDatabase?

Reply:
As of right now, no, the LBManager is the interface for basically everything. Perhaps we can revisit this later.